design/sketch.webidl

```diff
@@ -119,8 +111,31 @@ dictionary WebGPUTextureDescriptor {
     WebGPUTextureUsageFlags usage;
 };
 
+// Texture view
+enum WebGPUTextureViewDimension {
+    "1d",
+    "2d",
+    "2darray",
+    "cube",
+    "cubearray",
+    "3d"
+};
+
+dictionary WebGPUTextureViewDescriptor {
```

Comment:
we'll need aspects from #89  here so probably makes sense to wait for it first

Reply:
It sounds like we might want to use a special format for this as mentioned in #79 ?

Reply:
I think another choice is to add a flag to specify which part (color, depth or stencil) is covered in the texture view (like the aspectMask in VkImageSubresourceRange).